src/hooks/queries/useEvmFees.ts

```diff
@@ -0,0 +1,68 @@
+import type { ChainId } from '@shapeshiftoss/caip'
+import { skipToken, useQuery } from '@tanstack/react-query'
+import { useMemo } from 'react'
+import { reactQueries } from 'react-queries'
+import { selectEvmFees } from 'react-queries/selectors'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import {
+  assertGetEvmChainAdapter,
+  getFeesWithWallet,
+  type GetFeesWithWalletArgs,
+  isGetFeesWithWalletArgs,
+} from 'lib/utils/evm'
+import {
+  selectFeeAssetByChainId,
+  selectMarketDataByAssetIdUserCurrency,
+} from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+type UseEvmFeesProps = Omit<
+  GetFeesWithWalletArgs,
+  'wallet' | 'adapter' | 'accountNumber' | 'data'
+> & {
+  accountNumber: number | undefined
+  data: string | undefined
+  chainId: ChainId
+  enabled?: boolean
+  staleTime?: number
+  refetchInterval?: number | false | undefined
```

Comment:
```suggestion
  refetchInterval: number | false | undefined
```

Reply:
The rationale was it's up to the consumers to pass it or not, basically honoring `useQuery()`'s API for queryOptions.
But reading this again and checking LSP on this, since all current consumers *do* pass it (either as a number or `false`), there's probably value in enforcing it (if anything, as `undefined` if the consumer isn't concerned!

0ce565c90c also made its fren under required while at it!